flink-runtime/src/main/java/org/apache/flink/streaming/runtime/operators/sink/committables/CommittableCollectorSerializer.java

```diff
@@ -48,18 +48,21 @@ public final class CommittableCollectorSerializer<CommT>
     private static final int MAGIC_NUMBER = 0xb91f252c;
 
     private final SimpleVersionedSerializer<CommT> committableSerializer;
-    private final int subtaskId;
-    private final int numberOfSubtasks;
+    /** Default values are used to deserialize from Flink 1 that didn't store the information. */
+    private final int defaultSubtaskId;
+    /** Default values are used to deserialize from Flink 1 that didn't store the information. */
+    private final int defaultNumberOfSubtasks;
+
     private final SinkCommitterMetricGroup metricGroup;
 
     public CommittableCollectorSerializer(
             SimpleVersionedSerializer<CommT> committableSerializer,
-            int subtaskId,
-            int numberOfSubtasks,
+            int defaultSubtaskId,
+            int defaultNumberOfSubtasks,
```

Comment:
I find `default` the wrong word. Can we call it `currentTask[....]`

Reply:
Did you read the comment on the fields?

```
    /** Default values are used to deserialize from Flink 1 that didn't store the information. */
    private final int defaultSubtaskId;
    /** Default values are used to deserialize from Flink 1 that didn't store the information. */
    private final int defaultNumberOfSubtasks;
```

After the PR we don't need that information anymore and just used for backward compatibility. I actually thought of just setting them to some constant as it shouldn't matter but decided to emulate the old behavior as much as possible.

If you still think `currentSubtaskid` is more appropriate, how about `owningSubtaskId`?

Reply:
I like `owningSubtaskId`